ext/node/polyfills/vm.ts

```diff
@@ -25,8 +26,8 @@ export class Script {
     notImplemented("Script.prototype.runInContext");
   }
 
-  runInNewContext(_contextObject: any, _options: any) {
-    notImplemented("Script.prototype.runInNewContext");
+  runInNewContext(contextObject: any, _options: any) {
```

Comment:
Change to `options` and if it's not undefined print a warning that this option is currently not supported.